presentation/scripts/lab1-3.sh

```diff
@@ -17,12 +17,16 @@ WORKER_IP=$(cat out | grep publicIP | sed "s/.*\"\([0-9].*\)\".*/\1/g" )
 # Next we need the node port assignment of the applicationon the cluster.
 # It was automatically assigned by the kubernetes runtime"
 
-comment --nolf "Notice the 'publicIP' field"
+comment --pauseafter "Notice the 'publicIP' field"
+
 comment "Get the nodePort of the service"
 doit kubectl get svc hello-world -ojson
 
 SERVICE_PORT=$(cat out | grep nodePort | sed "s/.*: *\([0-9]*\).*/\1/g")
 
-comment --nolf "Notice the 'nodePort' field"
+comment --pauseafter "Notice the 'nodePort' field"
```

Comment:
Did not notice this until now. We are intentionally printing out the json.

Reply:
yea, is that a concern? I wanted to show the full output and then draw peole's attention to the nodePort.

Reply:
no issue, was one of the things I wanted to ask about.